templates/CRM/common/footer.tpl

```diff
@@ -14,18 +14,17 @@
   {/if}
 
   <div class="crm-footer" id="civicrm-footer">
-    {crmVersion assign=version}
-    {ts}Powered by CiviCRM{/ts} <a href="https://download.civicrm.org/about/{$version}" rel="external" target="_blank">{$version}</a>.
     {if $footer_status_severity}
-      <span class="status{if $footer_status_severity gt 3} crm-error{elseif $footer_status_severity gt 2} crm-warning{else} crm-ok{/if}">
+    <span class="status{if $footer_status_severity gt 3} crm-error{elseif $footer_status_severity gt 2} crm-warning{else} crm-ok{/if}">
       <a href="{crmURL p='civicrm/a/#/status'}">{$footer_status_message}</a>
-    </span>
+    </span>&nbsp;
+    {elseif call_user_func(array('CRM_Core_Permission','check'), 'administer CiviCRM')}
+    <span class="status crm-status-none">
+      <a href="{crmURL p='civicrm/a/#/status'}">System Status</a>
```

Comment:
please wrap the string in `{ts}System Status{/ts}` for translation

Reply:
Oops, done.